lib_xua/src/core/audiohub/xua_audiohub.xc

```diff
@@ -577,7 +576,9 @@ void testct_byref(chanend c, int &returnVal)
         returnVal = 1;
 }
 
-#if (XUA_DFU_EN == 1)
+#if (XUA_DFU_EN == 1) && ((NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0))
+/* This function is a dummy version of the deliver thread that does not
+   connect to the codec ports. It is used during DFU reset. */
 [[combinable]]
 static void dummy_deliver(chanend ?c_out, unsigned &command)
```

Comment:
I can add a timer. What speed? 192kHz?

I don't really know why it has to run more than once to be honest.. I know it would be required to catch a request that was in-flight.

Reply:
192k seems appropriate.  Haven't seen this issue for ages but it may add a bit of safety

Reply:
one issue I realised when adding a simple delay here, when combined with the DFUHandler task this will potentially slow down DFU requests. It could be done "properly" by adding a timer case in the select, but im a bit hesitant to make this larger change at this point.